cernopendata/modules/fixtures/data/records/cms-detailed-luminosity-information.json

```diff
@@ -0,0 +1,156 @@
+    },
+    "accelerator": "CERN-LHC",
+    "collaboration": {
+      "name": "CMS Collaboration",
+      "recid": "451"
+    },
+    "collections": [
+      "CMS-Luminosity-Information"
+    ],
+    "date_created": [
+      "2012"
+    ],
+    "date_published": "2020",
+    "distribution": {
+      "formats": [
+        "txt",
+        "csv"
+      ]
+    },
+    "doi": "FIXME2012",
+    "experiment": "CMS",
+    "files": [
+      {
+        "checksum": "FIXME",
+        "size": 99999,
+        "uri": "root://eospublic.cern.ch//eos/opendata/cms/luminosity/2012/2012lumi_pxl.txt"
+      },
+      {
+        "checksum": "FIXME",
+        "size": 99999,
+        "uri": "root://eospublic.cern.ch//eos/opendata/cms/luminosity/2012/2012lumi_hfoc.txt"
+      },
+      {
+        "checksum": "FIXME",
+        "size": 99999,
+        "uri": "root://eospublic.cern.ch//eos/opendata/cms/luminosity/2012/2012RunBlumi_pxl.txt"
+      },
+      {
+        "checksum": "FIXME",
+        "size": 99999,
+        "uri": "root://eospublic.cern.ch//eos/opendata/cms/luminosity/2012/2012RunBlumi_hfoc.txt"
+      },
+      {
+        "checksum": "FIXME",
+        "size": 99999,
+        "uri": "root://eospublic.cern.ch//eos/opendata/cms/luminosity/2012/2012RunClumi_pxl.txt"
+      },
+      {
+        "checksum": "FIXME",
+        "size": 99999,
+        "uri": "root://eospublic.cern.ch//eos/opendata/cms/luminosity/2012/2012RunClumi_hfoc.txt"
+      },
+      {
+        "checksum": "FIXME",
+        "size": 99999,
+        "uri": "root://eospublic.cern.ch//eos/opendata/cms/luminosity/2012/2012lumibyls_pxl.csv"
+      },
+      {
+        "checksum": "FIXME",
+        "size": 99999,
+        "uri": "root://eospublic.cern.ch//eos/opendata/cms/luminosity/2012/2012lumibyls_hfoc.csv"
+      }
+    ],
```

Comment:
The prescale2012.txt will appear in this list as well?

Reply:
I left it in the description with the link pointing to the original location because I did not want to duplicate it here  Shall I duplicate it anyway?  Likewise for the 2011 record.

Reply:
OK, now I see. Maybe then mention explicitly:  The change of prescales (run, lumi section, index of prescales) is recorded in <a href=\"/record/1051/files/prescale2011.txt\">prescale2011.txt</a> of the record  ... (but judge the readability, if it gets complicated to read, then not worth adding)